tests/PySys/tedge_mapper_c8y/tedge_mapper_c8y_positive/run.py

```diff
@@ -0,0 +1,77 @@
+from pysys.basetest import BaseTest
+
+import time
+import json
+
+"""
+Validate a tedge-mapper-c8y message that is published
+on c8y/measurement/measurements/create
+
+Given a configured system
+When we start the tedge-mapper-c8y systemctl service
+When we start tedge sub with sudo in the background
+When we publish a Thin Edge JSON message
+Then we kill tedge sub with sudo as it is running with a different user account
+Then we validate the JSON message in the output of tedge sub
+Then we stop the tedge-mapper-c8y systemctl service
+
+"""
+
+
+class TedgeMapperC8y(BaseTest):
+    def setup(self):
+        self.tedge = "/usr/bin/tedge"
+        self.sudo = "/usr/bin/sudo"
+
+        mapper = self.startProcess(
+            command=self.sudo,
+            arguments=["systemctl", "start", "tedge-mapper-c8y"],
+            stdouterr="tedge_mapper_c8y",
+        )
+
+        self.addCleanupFunction(self.mapper_cleanup)
+
+    def execute(self):
+        sub = self.startProcess(
+            command=self.sudo,
+            arguments=[self.tedge, "mqtt", "sub", "--no-topic", "c8y/measurement/measurements/create"],
```

Comment:
Why specifically this c8y/measurement/measurements/create topic?

Reply:
c8y mapper publishes on `c8y/measurement/measurements/create`.

Reply:
That is the topic (topic template (?)) where all the other c8y measurements go to.